src/hotspot/share/oops/klass.hpp

```diff
@@ -428,6 +433,7 @@ class Klass : public Metadata {
   static ByteSize next_sibling_offset()          { return byte_offset_of(Klass, _next_sibling); }
 #endif
   static ByteSize bitmap_offset()                { return byte_offset_of(Klass, _bitmap); }
+  static ByteSize misc_flags_offset()            { return byte_offset_of(Klass, _misc_flags); }
```

Comment:
```suggestion
  static ByteSize misc_flags_offset()            { return byte_offset_of(Klass, _misc_flags._flags); }
```

Reply:
We probably shouldn't assume the _flags field starts at offset 0.

Reply:
Yes, this was wrong.  Thanks for seeing this.